src/UI/Art/Animation/SimpleMoveAnimation.java

```diff
@@ -0,0 +1,93 @@
+package UI.Art.Animation;
+
+import java.awt.Graphics;
+import Engine.Path;
+import Engine.XYCoord;
+import Units.Unit;
+
+public class SimpleMoveAnimation implements GameAnimation
```

Comment:
Why "Simple"? Are you also going to make a `ComplexMoveAnimation`?

Reply:
Is teleportation not a form of movement?

Reply:
Depends on which sci-fi nerd you ask, but we'll just say yes. Moving along the ground is the default though, and naming the animation to match the action is convenient.
Teleportation is a way of relocating a unit, but nobody is going to assume teleportation if you say you want to "move a unit here" (unless that's the only/most reasonable way that can happen in context).
I mean, it works, it just a) doesn't match the action, and b) seems to point to a broader picture that doesn't exist.